.gitmodules

```diff
@@ -0,0 +1,8 @@
+[submodule "vendor/locales"]
+	path = vendor/locales
+	url = https://github.com/citation-style-language/locales.git
+	branch = master
```

Comment:
I think we should use a sha1 to have stable release

Reply:
I'm not sure if that is possible? As far as I know a sha1 is checked out. If you look at the vendor/locales or vendor/styles file a specific commit is mentioned. 
If I do a git pull on this repo, this will result in a change I need to commit for this repo.

Reply:
You're correct, nevermind, I forgot how submodules are working 😅 

> If I do a git pull on this repo, this will result in a change I need to commit for this repo.

👍🏻